content/webapp/utils/iiif/v3/index.ts

```diff
@@ -532,3 +534,36 @@ export function getCollectionManifests(manifest: Manifest): Canvas[] {
     .flat();
   return [...firstLevelManifests, ...collectionManifests] as Canvas[];
 }
+
+type CustomSpecificationBehaviors = SpecificationBehaviors & 'placeholder';
+// Whether something is born digital or not is determined at the canvas level within a iiifManifest
+// It is therefore possible to have a iiifManifest that contains:
+// - only born digital items
+// - no born digital items
+// - a mix of the two.
+// We need to know which we have to determine the required UI.
+export function getBornDigitalStatus(manifest: Manifest): BornDigitalStatus {
+  const hasBornDigital =
+    manifest?.items.some(
+      canvas =>
+        canvas?.behavior?.includes(
+          'placeholder' as CustomSpecificationBehaviors
```

Comment:
What is `placeholder` in this case? My knowledge of the manifests still being quite poor.

Reply:
It's a custom behavior value that alerts us to the fact that the canvas isn't the _real_ content. This is the value we'd get when something is born digital. [More in depth explanation is in the RFC](https://github.com/wellcomecollection/docs/blob/main/rfcs/046-born-digital-iiif/README.md)

Reply:
Ohhhh yes I'll have a good read-through of that! Thanks!